src/Services/GTT/shared/GTT.Domain/Entities/UploadImage.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GTT.Domain.Entities
+{
+    public class UploadImage
```

Comment:
Objects in GTT.Domain represents for tables in DB, this class is not for that purpose, so move it to GTT.Application